pkg/theiactl/commands/utils.go

```diff
@@ -48,10 +52,85 @@ func PolicyRecoPreCheck(clientset *kubernetes.Clientset) error {
 	if len(pods.Items) < 1 {
 		return fmt.Errorf("can't find the policy-reco-spark-operator Pod, please check the deployment of the Spark Operator")
 	}
-	// TODO: add check the ClickHouse deployment and ping ClickHouse DB after port forwarder finished
+	CheckClickHousePod(clientset)
 	return nil
 }
 
+func CheckClickHousePod(clientset *kubernetes.Clientset) error {
+	// Check the ClickHouse deployment in flow-visibility namespace
+	pods, err := clientset.CoreV1().Pods(flowVisibilityNS).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: "app=clickhouse",
+	})
+	if err != nil {
+		return fmt.Errorf("error %v when finding the ClickHouse Pod, please check the deployment of the ClickHouse", err)
+	}
+	if len(pods.Items) < 1 {
```

Comment:
Do you think it might make sense to verify pod status is running?

Reply:
Added.